vspace/src/main/webapp/WEB-INF/views/layouts/navbar.html

```diff
@@ -15,6 +15,17 @@
 					      	<div style="height: 5px; width: 5px; border-radius:50%; background-color: var(--success)"></div>
 					      	<em style="color: var(--dark-grey); font-size: 12px; margin-left: 10px">logged in as <span th:text="${#authentication.name}"></span></em>
 					    </div>
+		        	</li>
+		        	<a th:if="${aboutPageConfigured==true}"  th:href="@{/exhibit/about}">
+		        	<span th:if="${aboutPageConfigured==true}" class = "icon-info secondary" />     	
+	        		<span class="custom_Text"
+                                   th:if="${aboutPageConfigured==true}"
```

Comment:
Instead of have three if statements for each element I would simply surround all elements with a span and put the if in that span.